plasmapy/plasma/plasma_factory.py

```diff
@@ -1,13 +1,11 @@
-from plasmapy.plasma.plasma_base import GenericPlasma
-from plasmapy.utils.datatype_factory_base import (
-    BasicRegistrationFactory,
-    MultipleMatchError,
-    NoMatchError,
-    ValidationFunctionError,
-)
-
+"""
+Module for defining the frame work around the plasma factory.
```

Comment:
```suggestion
Module for defining the framework around the plasma factory.
```

Aaaand I'm officially in a Warcraft mood, the first time I posted this suggestion I accidentally typed "frame ork".